app/models/extractors/question_extractor.rb

```diff
@@ -19,8 +22,29 @@ def extract_data_for(classification)
 
     private
 
+    def fetch_annotations(classification)
+      case if_missing
+      when "error"
+        begin
+          classification.annotations.fetch(task_key)
+        rescue KeyError
+          raise MissingAnnotation, "No annotations for task #{task_key}"
+        end
+      when "ignore"
+        begin
+          classification.annotations.fetch(task_key)
+        rescue KeyError
+          []
+        end
+      end
+    end
+
     def task_key
       config.fetch("task_key", "T0")
     end
+
+    def if_missing
+      config["if_missing"] || "error"
```

Comment:
is the default needed if the presence validator is set? Probably if you don't backfil...